### PR TITLE
Organisation beta files

### PR DESCRIPTION
**Pull request by Jamie-Atkinson**
the files for the government organisation register beta environment.
There is a new datatype for the beta datatype register
all descriptions are approved by Gurinder

### CODE REVIEW

File: data/beta/register/government-organisation.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+fields:
+- government-organisation
+- name
+- website
+- start-date
+- end-date
+phase: beta
+register: government-organisation
+registry: government-digital-service
+text: 'Government department, agency or teams that exist on gov.uk'
```
Comment (michaelabenyohai) [inline data/beta/register/government-organisation.yaml]: GOV.UK needs to be capitalised

Comment (Jamie-Atkinson) [inline data/beta/register/government-organisation.yaml]: thanks, fixed

**Comment by Jamie-Atkinson** [comment]
@psd can you check and approve this please

File: data/beta/field/government-organisation.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+cardinality: '1'
+datatype: string
+field: government-organisation
+phase: beta
+register: government-organisation
+text: The unique code representing a government organisation.
```
Comment (psd) [inline data/beta/field/government-organisation.yaml]: "unique" is a tautology, we should remove that.

**Comment by psd** [comment]
Remove the superfluous word 'unique' otherwise looks good!

**Pull request merged**